contract-ui/tabs/overview/components/MarketplaceDetails.tsx

```diff
@@ -0,0 +1,366 @@
+        }))
+        .reverse() || [],
+    [directListingsQuery?.data],
+  );
+
+  const englishAuctions = useMemo(
+    () =>
+      englishAuctionsQuery?.data
+        ?.map<ListingData>((v) => ({
+          ...v,
+          sellerAddress: v.creatorAddress,
+          type: "english-auction",
+          currencyValue: v.buyoutCurrencyValue,
+        }))
+        .reverse() || [],
+    [englishAuctionsQuery?.data],
+  );
+
+  return (
+    <Flex gap={6} flexDirection="column">
+      <Heading size="title.sm">Listings</Heading>
+      <ListingStatsV3 contract={contract} />
+      {!directListingsQuery.isLoading && directListings.length === 0 ? null : (
+        <>
+          <Flex align="center" justify="space-between" w="full">
+            <Heading size="label.lg">Direct Listings</Heading>
+            <TrackedLink
+              category={trackingCategory}
+              label="view_all_direct_listings"
+              color="blue.400"
+              _light={{
+                color: "blue.600",
+              }}
+              gap={4}
+              href={directListingsHref}
+            >
+              View direct listings -&gt;
+            </TrackedLink>
+          </Flex>
+          <ListingCards
+            listings={directListings}
+            isLoading={directListingsQuery.isLoading}
+            trackingCategory={trackingCategory}
+            isMarketplaceV3
+          />
+        </>
+      )}
+      {!englishAuctionsQuery.isLoading &&
+      englishAuctions.length === 0 ? null : (
+        <>
+          <Flex align="center" justify="space-between" w="full">
+            <Heading size="label.lg">English Auctions</Heading>
+            <TrackedLink
+              category={trackingCategory}
+              label="view_all_english_auctions"
+              color="blue.400"
+              _light={{
+                color: "blue.600",
+              }}
+              gap={4}
+              href={englishAuctionsHref}
+            >
+              View english auctions -&gt;
+            </TrackedLink>
+          </Flex>
+          <ListingCards
+            listings={englishAuctions}
+            isLoading={englishAuctionsQuery.isLoading}
+            trackingCategory={trackingCategory}
+            isMarketplaceV3
+          />
+        </>
+      )}
+    </Flex>
+  );
+};
+
+const dummyMetadata: (idx: number) => ListingData = (idx) => ({
+  asset: {
+    name: "Loading...",
+    description: "lorem ipsum loading sit amet",
+    id: `${idx}`,
+    uri: "",
+  },
+  sellerAddress: `0x_fake_${idx}`,
+  type: idx % 2 === 0 ? "direct-listing" : "english-auction",
+  currencyValue: {
+    name: "Ethereum",
+    symbol: "ETH",
+    decimals: 18,
+    value: BigNumber.from("0"),
+    displayValue: "0",
+  },
+  startTimeInSeconds: BigNumber.from(idx),
+});
+
+interface ListingCardsProps {
+  listings: ListingData[];
+  isLoading: boolean;
+  trackingCategory: TrackedLinkProps["category"];
+  isMarketplaceV3?: boolean;
```

Comment:
this boolean should be isMarketplaceV1, that's the one we have a special case for, marketplace v3 is not a "prebuilt" per se but a combination of different contract extensions